trestle/core/commands/add.py

```diff
@@ -35,7 +35,12 @@
 
 
 class AddCmd(CommandPlusDocs):
-    """Add a subcomponent to an existing model."""
+    """Add an OSCAL component/subcomponent to the specified component.
+
+    This method takes input a filename and a list of comma-seperated element path. Element paths are field aliases.
```

Comment:
Just a couple examples here would help both users and coders.  'Element paths are field aliases' is concise and accurate but somewhat opaque.